docs/source/installation.rst

```diff
@@ -0,0 +1,110 @@
+.. _installation:
+
+Installation instructions
+=========================
+
+Note: T3 was only tested on Linux (Ubuntu_ 18.04.1 LTS) and Mac machines. We don't expect it to work on Windows.
+T3 can be installed on a server, as well as on your local desktop / laptop, submitting jobs to the server/s.
+Note that T3 requires installation of both `RMG <http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html>`_, Reaction Mechanism Generator, and `ARC <https://reactionmechanismgenerator.github.io/ARC/installation.html>`_, Automated Rate calculator.
+
+
+.. _path:
+
+Clone and setup path
+^^^^^^^^^^^^^^^^^^^^
+
+- Download and install the `Anaconda Python Platform`__ for Python 3.7 or higher if you haven't already.
+- Get git if you don't have it already by typing sudo apt-get install git in a terminal.
+- Clone T3's repository to by typing the following command in the desired folder (e.g., under `~/home/Code/`)::
+
+    git clone https://github.com/ReactionMechanismGenerator/T3.git
+			  
+- Add T3 to your local path in .bashrc (make sure to change `~/Path/to/T3/` accordingly)::
+
+    export PYTHONPATH=$PYTHONPATH:~/Path/to/T3/
+
+__ anaconda_
+
+
+Install dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+- Install the latest DEVELOPER version of RMG (which has Arkane).
```

Comment:
no need to install RMG for T3 (though the user may do so - perhaps we should have two env files, with and w/o `rmgpy`)

Reply:
I think including RMG in the environment file made sense. I just didn't pay attention before...